scripts/cloudhealth/README.adoc

```diff
@@ -0,0 +1,11 @@
+This script is used to configure cross-service permissions required to
+allow CloudHealth to collect cost-related data.
+
+It creates AWS IAM Role, Profile, and Role->Profile link. Then, it adds the AWS
+Account info to CloudHealth.
+
+Usage:
+
+        $ export CLOUDHEALTH_API_KEY='some-api-key'
+        $ add-billing-role.py -p AWS_PROFILE -k random-string -n role-name
```

Comment:
Can you define what "random-string" means.  What's it used for?  Why is it random?  Would you re-use one?  If it's truly random, then the script should generate it's own random data (e.g. openssl rand -hex 20) to protect against the user not providing "random enough" input.

Reply:
I think I meant to reference the "-e" flag, not "-k", but either way it can be removed.